internal/pkg/template/workload.go

```diff
@@ -593,6 +599,24 @@ type WorkloadOpts struct {
 	Subscribe *SubscribeOpts
 }
 
+// HealthCheckProtocol returns the protocol for the Load Balancer health check,
+// or an empty string if it shouldn't be configured, defaulting to the
+// target protocol. (which is what happens, even if it isn't documented as such :))
+// https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthcheckprotocol
+func (w WorkloadOpts) HealthCheckProtocol() string {
+	switch {
+	case w.HTTPHealthCheck.Port == "443":
+		return "HTTPS"
+	case w.HTTPTargetContainer.IsHTTPS() && w.HTTPHealthCheck.Port == "":
+		return "HTTPS"
```

Comment:
do we need this case then, or can it be ""? since the TG protocol will be set to HTTPS

Reply:
i think we should explicitly configure it since the behavior isn't documented correctly